app/src/main/java/com/example/fitjournal/MainActivity.kt

```diff
@@ -81,15 +84,17 @@ class MainActivity : ComponentActivity() {
                                     LibraryScreen(
                                         modifier = mainScreenModifier,
                                         libraryWorkoutState = libraryScreenViewModel.libraryWorkoutState,
-                                        displayBlur = { showChildFabs = it }
+                                        displayBlur = { showChildFabs = it },
+                                        isBlurActive = showChildFabs,
```

Comment:
Alex, if we show the blur when we show the Fab menu, why do we need displayBlur and isBlurActive?

Reply:
i was trying to leave your blur logic and add on top of it but let me see if i can remove it

Reply:
Removed it